contents/handbook/growth/sales/customer-onboarding.md

```diff
@@ -1,37 +1,116 @@
-  - Work with the client to make sure they:
-    - (i) track product performance (they've got events coming in, they are measuring AARRR and they've got a dashboard that breaks down each high level metric)
-    - (ii) diagnose performance
-    - (iii) release changes with our platform. Often customers may be using just one of the three areas at first.
-    - file issues for feature requests if we're missing something / talk to our product team about these
-
-There are regular checks you should do in bulk across all paying accounts:
-
-- Weekly check of usage/invoice amounts
-  - do any paying customers have unexpectedly large or small usage?
-  - any issues collecting payments?
-  - any cancellations? (gulp)
-- Every two weeks do a check of latest features - share cool stuff 1/1 with every customer in Slack.
-- Every month, get our product team to discuss potentially relevant roadmap features with them, or consider if our product marketing team should run a case study with our clients (free or paid). This helps deepen the relationships we have with them.
+We believe our unique value propositon is that we provide a fully-integrated Product Data stack, allowing users to go across Analytics, Recordings and Experimentation with ease.  To that end, we want them to see this value as quickly as possible when evaluating us against other solutions.
+
+For high-touch prospective customers the following process will get them onboarded quickly so that they can experience the value we provide using their own product data.
+
+The process should run for 2 weeks by default, but can be extended if we think it's worth the additional effort.
+
+The aim at the end of the evaluation is to have them:
+1. Sending in auto or custom captured event data
+2. Enabled session recordings
+3. Created a trend chart tracking User Acquisition
+4. Created a funnel tracking Activation
+5. Added the above to a dashboard
+
+### At the end of the demo call
+
+If at the end of a demo call we think a customer qualifies for high-touch onboarding we should outline our suggested evaluation approach.  If they aren't quite ready to kick the evaluation off then we should follow up with a templated email reminding them of the process, then check in with them after they've had some time to regroup.
+
+#### High touch criteria
+
+As a small team we have limited bandwidth to run customer evaluations so we need to focus on potential customers who:
+1. Are likely to contract above $20k with us.
+   (Ideally we qualify this by giving indicative pricing in the demo)
+2. Are likely to enter into an annual contract.
+   (This is quite a high-effort process for people just going month to month)
+3. Are ready to get hands-on with PostHog and will make a decision in weeks, not months.
+
+### Expectations of the customer
```

Comment:
Is this part of the demo call or the kick off call?

Reply:
More of a high level what we need them to do / we will tell them this beforehand